position_tools/core.py

```diff
@@ -15,6 +15,12 @@ def get_centroid(position1, position2):
     centroid_position : np.ndarray, shape (n_time, 2)
 
     """
+    if np.all(position1 == 0):
```

Comment:
I would use `np.allclose(position1, 0.0)`

Reply:
I'm not sure - I think the goal is to evaluate exactly zero as a special case that means null. Not ideal, but I think there's an upstream restriction on not using nulls. 

When I think of the 'close but not exactly equal' case, I assume that's actual data that stayed close to origin. Or are there other reasons we would get a small flutter in the null case?

Reply:
Yeah, that is true although conversely I don't think there are really cases where the position will be slightly around zero and not be this case (within the default tolerance bounds). I think the only control here is that floating point representations are variable.